{{cookiecutter.repostory_name}}/devops/vultr_scripts/vultr-deploy.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# deploy to list of IPs from `instances_ip.txt` (see `vultr-get-instances.py`)
+
+import subprocess
+from pathlib import Path
+
+pwd = Path(__file__).parent
+
+
+with open(pwd / "instances_ip.txt") as f:
+    ips = f.readlines()
+
+errs = []
+for ip in ips:
+    print('deploying to', ip)
+    try:
+        res = subprocess.Popen(
```

Comment:
Have you considered `multiprocessing.Pool`? It could probably yield better results over multiple machines.